packages/colorpicker/lib/src/state/position_fraction_state.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'position_fraction_state.g.dart';
+
+@riverpod
+class PositionFractionNotifier extends _$PositionFractionNotifier {
```

Comment:
This state can be tracked with ```PositionNotifier```aswell. I dont see why it would be necessary to create additional provider. Though I am also unsure why you are tracking a position in ```PositionNotifier```. Maybe someone that has already reviewed it might have a better insight (@msesko or @juliajulie95).

Reply:
Yes, you are right, the fraction can be tracked from the position itself.
The position basically determines where should the slider be placed on horizontal dragging or even a tap on the opacity bar. That's required I think?